docs/cloud/faq/sla.md

```diff
@@ -0,0 +1,23 @@
+# Service Level Agreement & Security
+
+## What are the SLA ?
+
+Read more in our [Service Level Agreement](https://www.eventstore.com/cloud-services-service-level-agreement) document
+
+## Does EventStore Ltd have security policy
+
+EventStore Ltd.  has a security policy.  
```

Comment:
```suggestion
EventStore Ltd. has a security policy.  
```

Reply:
Is the security policy publicly available?

Reply:
nope :-(

Reply:
Then we should mention that's not publicly available and the reasoning. If it's not available for the customer then from their perspective it doesn't exist.